### PR TITLE
added home screenshot in readme and added styling in home

### PR DESCRIPTION
**Pull request by vp4140**
Added the screenshot  of home in Readme and enhanced the CSS, page looks a bit visually appealing now, on all devices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,11 +56,15 @@ To contribute to hassle free, follow these steps:
...
-
 ---
 
 ## Getting Started
 
+## Glimpse of UI
+
+**HOME**
+![Screenshot](../master/home.png)
+
```
Comment (Sarthak-Mittal) [inline README.md:67]: please add this png in PR comments to preview

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,4 +1,7 @@
-<div class="jumbotron jumbotron-fluid my-jumbotron">
+<div
+  style="box-shadow: 10px 4px 40px 10px rgba(99, 99, 99, 0.47)"
```
Comment (Sarthak-Mittal) [inline src/app/app.component.html:2]: remove inline styling, add it to css file

**Review by Sarthak-Mittal — CHANGES_REQUESTED** [review]
please go through added comments

File: README.md (hunk 1 above)
Comment (vp4140) [reply README.md:67]: home

**Comment by Sarthak-Mittal** [comment]
Thanks @vp4140 !

**Pull request merged**